Mixpanel/Mixpanel.m

```diff
@@ -113,6 +113,17 @@ - (instancetype)initWithToken:(NSString *)apiToken launchOptions:(NSDictionary *
             [self trackPushNotification:remoteNotification event:@"$app_open"];
         }
 #endif
+        
+        dispatch_async(self.serialQueue, ^{
+            BOOL integrationTracked = [[NSUserDefaults standardUserDefaults] boolForKey:@"tracked_integration"];
```

Comment:
What would the repercussions be of moving the `if` statement outside of the `dispatch_async` block? There might not be a need to dispatch if we've already tracked.